seahorn/lib/nd_smack.c

```diff
@@ -44,6 +45,16 @@ void __SEA_assume(bool pre_cond) {
   __VERIFIER_assume(pre_cond);
 }
 
+void memhavoc(void *ptr, size_t size) {
```

Comment:
this is super expensive. It would be unfair to include this for smack

Reply:
Sorry I got lost from here. How do we perform `memhavoc` in Seahorn? Do all initialized values are non-determined?

Reply:
Maybe one simple way is using`memset`, but I'm not sure is it the same behavior we did in Seahorn.